ReactSkia/components/RSkComponentTextInput.cpp

```diff
@@ -168,6 +168,10 @@ void RSkComponentTextInput::onHandleKey(rnsKey eventKeyType, bool *stopPropagati
     textInputMetrics.contentSize.height = paragraph_->getHeight();
     textInputEventEmitter->onFocus(textInputMetrics);
     isInEditingMode_ = true;
+    if (!caretHidden_) {
+      std::cout<<"after onFocus!!!!!!!!!"<<std::endl;
```

Comment:
if the logging message is required , please use RNS_LOG_*
Same comment for the below log messages.